tests/test_exit_codes.py

```diff
@@ -2,12 +2,25 @@
 # pretty good. That plus accessing a few attributes will verify that it's
 # cross-platform.
 
+import os
 import exit_codes
 
 
 def test_ok():
-    exit_codes.ExitCode.OK
+    return exit_codes.ExitCode.OK
```

Comment:
Why does this need to return anything?

Reply:
It's just to prevent the PyCharm linter complaining about pointless code.

Reply:
`_ = exit_codes.ExitCode.OK` would also serve the same purpose.

Reply:
I'm fine with the return. Just wanted to make sure it had a reason.